pkg/plugin/manager/manager.go

```diff
@@ -75,12 +79,20 @@ func (m *manager) RunMigrations() error {
 	if len(paths) < 1 {
 		return nil
 	}
+
 	// Init directory for temporary copies of migrations
 	err = m.setupMigrationEnv()
 	if err != nil {
 		return err
 	}
 	defer m.cleanUp()
+
+	// First run the public schema migrations
+	err = m.runPublicMigrations()
+	if err != nil {
+		return err
```

Comment:
and I recognize we're not doing this in many places, but it'd be cool to decorate the error here, e.g. `fmt.Errorf("could not run public migrations: %w", err)`

incrementally avoiding `exit status 1` ftw 💪

Reply:
👍